testsuite/tests/prometheus/apicast/test_metrics.py

```diff
@@ -58,28 +59,36 @@ def test_metrics_from_target_must_contains_apicast_metrics(expected_metric, metr
 
 # there is certain delay before all appears in Prometheus
 @backoff.on_predicate(backoff.fibo, lambda x: sorted(x.keys()) != STATUSES, max_tries=8, jitter=None)
-def apicast_status_metrics(prometheus):
+def apicast_status_metrics(prometheus, container):
     """Reliable gathering of prometheus metrics
 
     Metrics in prometheus appear with some delay, therefore retry is needed
     to ensure expected values are available"""
     hits = prometheus.get_metric("apicast_status")
 
     return {
-        status: int(hit["value"][1]) for status in STATUSES for hit in hits if int(hit["metric"]["status"]) == status}
+        status: int(hit["value"][1]) for status in STATUSES
+        for hit in hits if int(hit["metric"]["status"]) == status and
```

Comment:
this indentation looks weird, I am surprised pylint doesn't complain (I do), I'd expect one level right (unless linters complain as they are more authoritative than I)
Also `and` at the beginning of line is much better than at the end.

(Btw I don't like this construct at all, however I already accepted it so not complaining here and now)

Reply:
Linters are complaining, It does not look right for me neither. I tried all permutations of one indentation on 1st, 2nd or on both lines at the same time, still not satisfied.